app/main.py

```diff
@@ -1,7 +1,7 @@
 def copy_file(command: str) -> None:
 
-    todolist = command.split()
-    cmd, file_name, file_copy_name = todolist
+    parts = command.split()
+    cmd, file_name, file_copy_name = parts
```

Comment:
if len(parts) less then 3, code will be broke here. add an additional check before